File: app/src/main/java/ru/samtakoy/core/services/api/pojo/RemoteFile.java
package ru.samtakoy.core.services.api.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class RemoteFile {


    @SerializedName("mPath")
    @Expose
    private String mPath;
    @SerializedName("mVersion")
    @Expose
    private Integer mVersion;
    @SerializedName("anchor")
    @Expose
    private Integer mAnchor;

    public String getPath(){ return mPath; }
    public Integer getVersion() { return mVersion; }
    public Integer getAnchor() { return mAnchor; }
}

File: app/src/main/java/ru/samtakoy/core/data/network/pojo/RemoteFile.kt
package ru.samtakoy.core.data.network.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class RemoteFile(
    @SerializedName("mPath")
    @Expose
    val path: String? = null,
    @SerializedName("mVersion")
    @Expose
    val version: Int? = null,
    @SerializedName("anchor")
    @Expose
    val anchor: Int? = null
)
